stable/insights-agent/values.yaml

```
@@ -136,15 +136,16 @@ releasewatcher:
 
 trivy:
   enabled: true
-  namespaceBlacklist: nil
+  namespaceBlacklist: []
   schedule: "rand */3 * * *"
   privateImages:
     dockerConfigSecret: ""
   maxConcurrentScans: 5
+  maxScansPerRun: 30
```

```suggestion
  maxScansPerRun: 10
```

I think that'll still take ~5 minutes